src/test/java/LoginTest.java

```diff
@@ -1,18 +1,40 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-public class LoginTest {
-    private static final Logger logger = LoggerFactory.getLogger(LoginTest.class);
+import static org.example.Constants.HOME_URL;
+
+public class LoginTest extends BaseTest {
+    private static final Logger log = LoggerFactory.getLogger(LoginTest.class);
+
+    @BeforeMethod
+    public void setUp() {
+        driver.get(HOME_URL);
+    }
+    @AfterSuite
+    public void closeDriver(){driver.quit();}
 
 
     @Test
-    public void loginTest() {
-        logger.info("Starting test...");
+    public void login() {
+        log.info("Start!!!");
+
+        cookieWindowService.cookieClickOk();
+        log.info("Click cookie OK");
 
+        log.info("Start WAIT");
+        advertisingWindowService.waitingForAdvertisement();
+        log.info("End WAIT");
 
+        mainHeaderService.clickToButtonSignIn();
+        log.info("Click button sign in");
 
-        logger.info("Test completed!");
+        loginPageService.enterLogin().enterPassword().clickButtonSignIn();
+        log.info("Sign in");
 
+        Assert.assertTrue(mainHeaderService.isLoggedIn(), "Login FAILED!");
```

Comment:
in negative case it will fail by Nosuchelementexeption, not by a assertion
<img width="1422" alt="Screenshot 2023-12-13 at 03 56 52" src="https://github.com/faa92/Kufar/assets/82497492/80865f84-7b2a-407c-9392-7be0b01bb920">